xclim/indices/_conversion.py

```diff
@@ -1006,14 +1008,16 @@ def longwave_upwelling_radiation_from_net_downwelling(
 
     Returns
     -------
-    xr.DataArray, [W m-2]
+    xr.DataArray, [same as inputs]
         Surface upwelling thermal radiation (rlus).
     """
-    rls = convert_units_to(rls, rlds)
+    if rls.units != rlds.units:
+        logging.warning(
+            "Radiation units are not synonymous. Units conversion performed."
+        )
+        rls = convert_units_to(rls, rlds)
```

Comment:
```suggestion
    rls = convert_units_to(rls, rlds)
```

We haven't done the warning yet in other indices. I think users _expect_ xclim to convert the differing units. Also, a `convert_units_to` call with identical units should be free!

Reply:
Valid. Let me revise this PR.